packages/search/tests/integration/components/navi-search-result/asset-test.js

```diff
@@ -0,0 +1,50 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Service from '@ember/service';
+import { set } from '@ember/object';
+
+module('Integration | Component | navi-search-result-asset', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  let service;
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    service = this.owner.lookup('service:navi-search-provider');
+    const store = this.owner.lookup('service:store'),
+      mockAuthor = store.createRecord('user', { id: 'ciela' });
+    this.owner.register(
+      'service:user',
+      Service.extend({
+        getUser: () => mockAuthor
+      })
+    );
+  });
+
+  test('it renders', async function(assert) {
```

Comment:
Does the asset search result component show any sort of message when no results are found? It should probably say `No results.` or something like that and we could test that that message is shown in this test.

Reply:
From the previous PR the search provider service will not return a result object if it doesn't contain data (https://github.com/yahoo/navi/blob/master/packages/search/addon/services/navi-search-provider.js#L51), so there won't be the case were we instantiate this component if it doesn't have data. The `No results` message will be show by the search bar component that is going to use the search provider service.

Reply:
Then it sounds like this test should be deleted